### ISSUE TITLE
makeWAR fails with UseMethod error

### ISSUE THREAD

**Issue by davidbmitchell** [beanumber/openWAR#81]
I try to run ```makeWAR()``` and it fails with a ```useMethod``` error.

```r
ds <- makeWAR(MLBAM2015)

Error in UseMethod("getModelRunExpectancy") : 
  no applicable method for 'getModelRunExpectancy' applied to an object of class "c('tbl_df', 'tbl', 'data.frame')"
```
Could it be related to changes made in commit c096eda6f270fa934610e6a2946d29c19526c0eb from 5 days ago?

**Comment by davidbmitchell** [comment]
Okay so I tried rerunning ```makeWAR()``` with the ```MLBAM2015``` data from the ```openWARData``` package, and it worked fine.

Here is what I ran originally:
```r
MLBAM2015 <- getData(start="2015-04-05", end="2015-06-21")
```
I just reran a version of the above code and it worked again fine. I'm going to run the full thing one more time to see if the error comes up again. I don't have a clue what happened.

**Comment by davidbmitchell** [comment]
I tried ```MLBAM2015 <- getData(start="2015-04-05", end="2015-06-21")``` again, and the ```useMethod``` occurred this time as well. 

As far as I can tell there are differences in the classes from the ```MLBAM2015``` I downloaded and the ```MLBAM2015``` from ```openWARData```.

The ```MLBAM2015``` I downloaded from above:
```r
class(MLBAM2015)
[1] "GameDayPlays" "grouped_df"   "tbl_df"       "tbl"          "data.frame"  
```
And the ```MLBAM2015``` from ```openWARData```
```r
class(MLBAM2015)
[1] "GameDayPlays" "data.frame"  
```

**Comment by davidbmitchell** [comment]
I found where the issue arises. It is in the ```getData()``` function. In commit c096eda6f270fa934610e6a2946d29c19526c0eb only one line was changed in the ```getData()``` function. On Line 59 it was originally:
```r
class(out) <- c("GameDayPlays", "data.frame")
```
Then it was changed to:
```r
 class(out) <- c("GameDayPlays", class(out))
```
Because we are using ```dplyr``` functions, these classes: ```c(""grouped_df", "tbl_df", "tbl")``` are being added to the classes of the ```out``` object. I figured this out by changing ```getData()``` back to the old version, but I don't think that is necessary. We just have to change ```getModelRunExpectancy``` method so that it can work with objects of the above classes. 

Would we change the ```setClass("GameDayPlays", contains = "data.frame")``` in ```GameDayPlays.R``` or would we have make a change in ```getModels.R```?
